### PR TITLE
rename config/secrets.yml -> config/secrets.yml.ci and configured it

### PR DESCRIPTION
**Pull request by mrJmek**
- [ ] @vovka 
- [x] @brouhaha88 
- [ ] @anatoliiibrahimov

### CODE REVIEW

**Comment by vovka** [comment]
Please, add two more reviewers

File: config/secrets.yml.ci [hunk 1]
```diff
@@ -10,11 +10,8 @@
...
 
-development:
-  secret_key_base: 8c3a3f4df071de9f80f48b9d67663b7226876d98fd26f784a46e146883a2f0814394f8c83e8df6f1c1debaeb245fc99174530203850e380042fa47e0a09738b7
-
 test:
-  secret_key_base: ad31f412429589fe94773bdb194c7c52e07634e28c17aff9efb1057e05951ae0d01eb07dc34b726e58a0ef495d7045142aff6c59af243b967372df51e649b55b
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
```
Comment (vovka) [inline config/secrets.yml.ci]: Do we really need production sections on CI?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-[![Coverage Status](https://coveralls.io/repos/github/superNazo/video_quiz/badge.svg?branch=master)](https://coveralls.io/github/superNazo/video_quiz?branch=master)
-[ ![Codeship Status for superNazo/video_quiz](https://codeship.com/projects/5c5eef50-c777-0133-7cf7-6e2d04338f84/status?branch=master)](https://codeship.com/projects/139036)
+[![Coverage Status](https://coveralls.io/repos/github/superNazo/video_quiz/badge.svg?branch=master&)](https://coveralls.io/github/superNazo/video_quiz?branch=master) [![Codeship Status for superNazo/video_quiz](https://codeship.com/projects/5c5eef50-c777-0133-7cf7-6e2d04338f84/status?branch=master)](https://codeship.com/projects/139036)
```
Comment (vovka) [inline README.md]: Why did you change this?

**Comment by vovka** [comment]
Please, add two more reviewers

File: README.md (hunk 2 above)
Comment (vovka) [inline README.md]: Oh, it was me)) Why my commit is here then?

File: config/secrets.yml.ci (hunk 1 above)
Comment (mrJmek) [inline config/secrets.yml.ci]: We could use exactly the same .yml.ci file (or create another with env. variables for production) for heroku. Codeship will send bash command (after successful) for heroku to rename .yml.ci to .yml

File: README.md (hunk 2 above)
Comment (mrJmek) [inline README.md]: If you would check another branches - your filed added to all branches that pulled last updates from master because today I revert my pull request #15

Comment (vovka) [inline README.md]: My commit definitely shouldn't be here.

File: config/secrets.yml.ci (hunk 1 above)
Comment (vovka) [inline config/secrets.yml.ci]: If this file called 'ci' then it should be used for CI. If it is used for heroku then it should reflect this fact in the name. Shouldn't it?

File: README.md (hunk 2 above)
Comment (mrJmek) [inline README.md]: If you will check all other branches - you could notice, that this file added to all other branches too.

File: config/secrets.yml.ci (hunk 1 above)
Comment (mrJmek) [inline config/secrets.yml.ci]: Agree with that.
I prefer to use one .yml.config file for both CI and heroku instead of having two same files with different names like .yml.ci and .yml.heroku (because all variable in those files would be the same except environment line - test or production)

How do you think?

File: README.md (hunk 2 above)
Comment (mrJmek) [inline README.md]: HOW TO RESOLVE IN OTHER BRANCHES?

git pull origin master
git push origin BRANCH_NAME

File: config/secrets.yml.ci (hunk 1 above)
Comment (mrJmek) [inline config/secrets.yml.ci]: Small research:
RubyGems.rog
https://github.com/rubygems/rubygems.org/tree/master/config
GitLabHQ
https://github.com/gitlabhq/gitlabhq/tree/master/config
OneBody all of them are using on file .yml.example 
https://github.com/churchio/onebody/blob/master/config/secrets.yml.example

Comment (brouhaha88) [inline config/secrets.yml.ci]: We are renaming this file in the test line of our Ci, but what for?
The builds are passing now with keys in our environment, so i don't see a need for the MV command and different file naming.

As of my understanding Genadij, you have fixed the issue of keys not working from the environment variables, meaning there is no more sense in renaming the file from xxx.yml.ci to xxx.yml.

This means leaving yml file like so `secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>` should do the trick.

Comment (vovka) [inline config/secrets.yml.ci]: Makes sense.

Comment (vovka) [inline config/secrets.yml.ci]: Genadij? Who is Genadij?:)
